### DISCUSSION TITLE
Lookup fields 👀

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#3645] [Ideas]
## Scope & Context
We need to introduce a new field type to display one-to-many relationships on a list view, we'll call this "Records Lookup"
We can start with displaying the list of people attached to each company as chips on the companies lists page.

## Related
https://github.com/twentyhq/twenty/issues/2213 >  "Field lookup" is a similar concept but probably different implementation because in the latter we only want to retrieve the first item 
https://github.com/twentyhq/twenty/issues/2219 > default way to display a record

**Comment by BOHEUS** [comment]
<img width="1680" height="1014" alt="image" src="https://github.com/user-attachments/assets/91f455c1-5b18-41d8-958e-d500931d2471" />
